src/components/show_page.js

```diff
@@ -16,9 +16,10 @@ export default class ShowPage extends React.Component {
   }
 
   componentDidMount() {
-    if (this.id) {
+    if (this.id !== undefined) {
       let url = `${process.env.REACT_APP_SERVER_URL}/api/v1/cocktails/${this.id}`
       axios.get(url, {cancelToken: this.source.token, params: {auth_token: Cookies.get('authToken')}}).then((res) => {
+        console.log(res);
```

Comment:
what is res?

Reply:
the response from the API call to the backend